### DISCUSSION TITLE
cta_stk_bk回测的胜率的分析结果似乎有问题

### DISCUSSION THREAD

**Discussion by tomasky** [wondertrader/wtpy#75] [Q&A] [answered]
如果设置    engine.configBacktest(202103010930,202103211500)
有１５个交易日，但是我只在其中的４个交易日有买卖操作，其他交易日没有任何买卖

但是　胜率统计的是   交易胜利天数／交易总天数，　这个结果就是胜率很低。

**Comment by wondertrader** [comment, marked as answer]
funds就是按照交易日统计的，有针对交易日的绩效分析，也有针对单笔的绩效分析，你仔细看看里面的指标
